inc/class-manage-multisite.php

```diff
@@ -0,0 +1,184 @@
+<?php
+
+defined( 'ABSPATH' ) || die( 'File cannot be accessed directly' );
+/**
+ *
+ */
+class Manage_Multisite {
```

Comment:
Let's call this PMProMultisiteMembership or PMPro_Manage_Multisite... something with a PMPro prefix. We could use namespaces, but I'd like to avoid that if possible.